science/gmtsar-data/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar-data
+version             1.0
+categories          science
+platforms           darwin
+license             GPL-3
+
+maintainers         ucsd.edu:dsandwell \
+                    ucsd.edu:xix016 \
+                    ucsd.edu:jdesanto \
```

Comment:
Please add a GitHub handle to each one, if available, in the form `{obfuscated:email @githubhandle}`.